spec/features/5_photos_spec.rb

```diff
@@ -1,120 +1,126 @@
+      end
+    end
 
-    within("nav") {
-      expect(page).to have_css(".fa-sign-out")
-    }
-  end
+    scenario "header uses Font Awesome fa-sign-out icon", points: 1 do
+      user = FactoryGirl.create(:user)
+      login_as(user, :scope => :user)
 
-  scenario "/photos lists comments with authors", points: 1 do
-    user_1 = FactoryGirl.create(:user, :username => "1", :email => "1@m.com")
-    user_2 = FactoryGirl.create(:user, :username => "2", :email => "2@m.com")
-    user_3 = FactoryGirl.create(:user, :username => "Three", :email => "three@m.com")
-    photo_1 = FactoryGirl.create(:photo, :user_id => user_3.id)
-    photo_2 = FactoryGirl.create(:photo, :user_id => user_3.id)
-    photo_3 = FactoryGirl.create(:photo, :user_id => user_3.id)
-    comment_1 = FactoryGirl.create(:comment, :user_id => user_1.id, :body => "comment_1", :photo_id => photo_1.id)
-    comment_2 = FactoryGirl.create(:comment, :user_id => user_2.id, :body => "comment_two", :photo_id => photo_3.id)
-    login_as(user_3, :scope => :user)
+      visit "/photos"
 
-    visit "/photos"
+      within("nav") {
+        expect(page).to have_css(".fa-sign-out")
+      }
+    end
 
-    expect(page).to have_content(comment_1.body)
-    expect(page).to have_content(comment_1.user.username)
-    expect(page).to have_content(comment_2.body)
-    expect(page).to have_content(comment_2.user.username)
-  end
+    scenario "/photos lists comments with authors", points: 1 do
+      comment_1 = FactoryGirl.create(:comment, :user_id => @user_1.id, :body => "comment_1", :photo_id => @photo_1.id)
+      comment_2 = FactoryGirl.create(:comment, :user_id => @user_2.id, :body => "comment_two", :photo_id => @photo_3.id)
+      login_as(@user_3, :scope => :user)
 
-  scenario "/photos shows Font Awesome heart icons to add/delete likes", points: 1, hint: h("font_awesome_css_must_match") do
-    user_1 = FactoryGirl.create(:user, :username => "1", :email => "1@m.com")
-    user_2 = FactoryGirl.create(:user, :username => "2", :email => "2@m.com")
-    photo_1 = FactoryGirl.create(:photo, :user_id => user_2.id)
-    photo_2 = FactoryGirl.create(:photo, :user_id => user_2.id)
-    like_1 = FactoryGirl.create(:like, :user_id => user_1.id, :photo_id => photo_1.id)
-    like_2 = FactoryGirl.create(:like, :user_id => user_2.id, :photo_id => photo_2.id)
-    login_as(user_1, :scope => :user)
+      visit "/photos"
 
-    visit "/photos"
+      photos = Photo.all
+      photos.each do |photo|
+        photo.comments.each do |comment|
+          expect(page).to have_content(comment.body)
+          expect(page).to have_content(comment.user.username)
+        end
+      end
+    end
 
-    expect(page).to have_css(".fa-heart")
-    expect(page).to have_css(".fa-heart-o")
-  end
+    scenario "/photos shows Font Awesome heart icons to add/delete likes", points: 1, hint: h("font_awesome_css_must_match") do
```

Comment:
We should drop the Font Awesome dependency and switch the tests and target to use copy "Like" / "Unlike".